datasource/src/main/scala/quasar/blobstore/azure/json.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2014–2018 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.blobstore.azure
+
+import argonaut._, Argonaut._, ArgonautRefined._
+
+object json {
```

Comment:
Shouldn't all of these be in companion objects?

Reply:
Yeah, on the one hand I'd like to have all the implicits automatically available when the domain class is imported, but on the other hand I also like to keep the domain code clean.

So far from what I have encountered I like something like this setup best (still a compromise though):
```
class MyDomainClass {
...
}

object MyDomainClass extends MyDomainClassImplicits
```

where `MyDomainClassImplicits` is then placed in a different file.

But in this case there's a bunch of small, separate classes in `config.scala` and it seems like this setup doesn't work as well. So I chose for a setup which requires the extra import to get access to the implicits. Would be interested in having something like above though.